### PR TITLE
[DF] Rename MakeColumnReaders => GetColumnReaders

### PR DESCRIPTION
**Pull request by eguiraud**
With the latest refactorings, MakeColumnReaders does not actually
create the column readers but simply gets them from RLoopManager or RColumnRegister.

### CODE REVIEW

File: tree/dataframe/inc/ROOT/RDF/ColumnReaderUtils.hxx [hunk 1]
```diff
@@ -75,24 +75,24 @@ struct RColumnReadersInfo {
 /// Create a group of column readers, one per type in the parameter pack.
 template <typename... ColTypes>
 std::array<std::shared_ptr<RDFDetail::RColumnReaderBase>, sizeof...(ColTypes)>
-MakeColumnReaders(unsigned int slot, TTreeReader *r, TypeList<ColTypes...>, RColumnReadersInfo &colInfo,
-                  const std::string &variationName = "nominal")
+GetColumnReaders(unsigned int slot, TTreeReader *r, TypeList<ColTypes...>, const RColumnReadersInfo &colInfo,
```
Comment (ikabadzhov) [inline tree/dataframe/inc/ROOT/RDF/ColumnReaderUtils.hxx:78]: I checked the body of this function. We take some class members (`fColNames`, `fLoopManager`, `fColRegister`) from the `colInfo` object and then pass them to `GetColumnReader`.

Afair `GetColumnReader` modifies these, i.e. the `std::move(treeColReader)`. Hence `RColumnReadersInfo` should not be `const` (hence windows does not complain)?

Comment (eguiraud) [reply tree/dataframe/inc/ROOT/RDF/ColumnReaderUtils.hxx:78]: The `treeColReader` in `std::move(treeColReader)` is a local variable created in `GetColumnReader`, so it's unrelated to the constness of `RColumnReadersInfo`.

I don't see where windows complains about violating constness either?

**Comment by eguiraud** [comment]
@phsft-bot build please

**Review by ikabadzhov — APPROVED** [review]
lgtm and jenkins is happy

**Pull request merged**